cli/src/pixl_cli/_io.py

```diff
@@ -138,30 +144,63 @@ def _load_parquet(
     return messages_df
 
 
-def _check_and_parse_parquet(private_dir: Path, public_dir: Path) -> pd.DataFrame:
-    for d in [public_dir, private_dir]:
-        if not d.is_dir():
-            err_str = f"{d} must exist and be a directory"
-            raise NotADirectoryError(err_str)
+def _parse_parquet(private_dir: Path, public_dir: Path) -> pd.DataFrame:
+    """Open parquet files and return joined data."""
+    parquet_dataset_dirs = [
+        public_dir / "procedure_occurrence",
+        private_dir / "person_links",
+        private_dir / "procedure_occurrence_links",
+    ]
+
+    if all(d.exists() and d.is_dir() for d in parquet_dataset_dirs):
+        messages_df = _parse_parquet_dataset(private_dir, public_dir)
```

Comment:
What happens if any of `parquet_dataset_dirs` are empty? Presumably `ds.dataset()` will throw an error? Might be worth adding a test to check that assumption?